recipes/openfaas.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: kbrew
+app:
+  args:
+    functionNamespace: openfaas-fn
```

Comment:
How do you override these at runtime?

Reply:
We are already working on this :) 
https://github.com/kbrew-dev/kbrew/issues/122